src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx

```diff
@@ -230,6 +241,127 @@ export const TransactionRow: React.FC<TransactionRowProps> = ({
     select: data => data?.unwrap(),
   })
 
+  const estimateFeesArgs = useMemo(() => {
+    if (!assetId || !wallet || !asset || !poolAsset) return undefined
+
+    const thorchainNotationAssetId = assetIdToPoolAssetId({
+      assetId: isRuneTx ? poolAsset.assetId : asset.assetId,
+    })
+    const transactionType = getThorchainLpTransactionType(asset.chainId)
+    const amountCryptoBaseUnit = toBaseUnit(amountCryptoPrecision, asset.precision)
+
+    switch (transactionType) {
```

Comment:
Can we use `estimateFeesArgs` instead of duplicating this logic?

Reply:
Discussed this one out, we can't do this unfortunately as `estimateFees()` expects a conssitent args shape, but the various Tx building utils don't